bower.json

```diff
@@ -12,18 +12,18 @@
     "url": "git://github.com/hdgarrood/purescript-benchotron.git"
   },
   "dependencies": {
-    "purescript-arrays": "~0.4.3",
-    "purescript-exists": "~0.2.0",
-    "purescript-identity": "~0.4.1",
-    "purescript-profunctor": "~0.3.1",
-    "purescript-strings": "~0.7.0",
-    "purescript-quickcheck": "~0.12.0",
-    "purescript-transformers": "~0.8.2",
-    "purescript-foldable-traversable": "~0.4.1",
-    "purescript-exceptions": "~0.3.1",
-    "purescript-node-fs": "~0.9.1",
-    "purescript-node-buffer": "~0.2.0",
-    "purescript-node-readline": "~0.3.1",
-    "purescript-datetime": "~0.9.0"
-  }
-}
+    "purescript-arrays": "^1.0.0",
+    "purescript-exists": "~1.0.0",
```

Comment:
I wouldn't go as far as "frowned upon", and they definitely don't *need* to be replaced with caret ranges in all instances, but in general they are a better default since they allow minor level changes. For most libraries, I think we definitely do want caret ranges instead of tilde ones.